google-cloud-vision/lib/google/cloud/vision/service.rb

```diff
@@ -66,9 +77,13 @@ def inspect
         protected
 
         def execute
+          require "grpc" # Ensure GRPC is loaded before rescuing exception
           yield
-        rescue Google::Apis::Error => e
+        rescue GRPC::BadStatus => e
```

Comment:
I was trying to be cautious, but not rescuing `GRPC::BadStatus` would actually simplify things a bit. Updated.